src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeCloudExceptionTests.cs

```diff
@@ -20,9 +20,12 @@ namespace Microsoft.Azure.Commands.Compute.Test.ScenarioTests
 {
     public class ComputeCloudExceptionTests
     {
+        ServiceManagemenet.Common.Models.XunitTracingInterceptor logger;
```

Comment:
ouch! are you aware that "ServiceManagemenet" is misspelled? 

I think it's been that way for a while :cry: